bluepysnap/edges.py

```diff
@@ -182,6 +122,8 @@ def properties(self, edge_ids, properties):
         ids = self.ids(edge_ids)
         # TODO : remove this due to the addition of ids to the EdgePopulation
```

Comment:
Is this TODO actual?

Reply:
Yes it is related to the Deprecate below.